scripts/save_existing_orgs_to_mongodb.pl

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/perl -w
+
+# This file is part of Product Opener.
+#
+# Product Opener
+# Copyright (C) 2011-2023 Association Open Food Facts
+# Contact: contact@openfoodfacts.org
+# Address: 21 rue des Iles, 94100 Saint-Maur des Fossés, France
+#
+# Product Opener is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use Modern::Perl '2017';
+use utf8;
+use Storable qw(lock_retrieve);
+use MongoDB;
+use Encode;
+
+my $database = "off";
+my $collection = "orgs";
+
+my $orgs_collection = MongoDB::MongoClient->new->get_database($database)->get_collection($collection);
+
+my $start_dir = $ARGV[0];
+
+if (not defined $start_dir) {
+    print STDERR "Pass the root of the organization directory as the first argument.\n";
+    exit();
+}
+
+sub retrieve {
+    my $file = shift;
+    if (!-e $file) {
+        return;
+    }
+    my $return = undef;
+    eval { $return = lock_retrieve($file); };
+    return $return;
+}
+
+my @orgs = ();
+
+sub find_orgs {
+    my ($dir, $code) = @_;
+    my $dh;
+
+    opendir $dh, "$dir" or die "could not open $dir directory: $!\n";
+    foreach my $file (sort readdir($dh)) {
+        chomp($file);
+        if ($file =~ /^(([0-9]+))\.sto/) {
```

Comment:
This looks for filenames that are of the form [some number].sto. This works well for barcodes, but orgs can have all kinds of characters.